src/app/features/notification/components/dialog-with-actions/dialog-with-actions.component.ts

```diff
@@ -23,6 +21,10 @@ export class DialogWithActionsComponent {
         this.closeWith(this.data.declineData);
     }
 
+    showDecline(): boolean {
+        return (this.data['hideDecline'] === undefined) ? true : !this.data.hideDecline;
```

Comment:
why are you using two different notation: `this.data['hideDecline']`  vs  `this.data.hideDecline`